.github/workflows/int_test.yml

```diff
@@ -84,35 +95,49 @@ jobs:
       CUSTOM_LOCATIONS_OID: ${{ inputs.custom-locations-oid }}
       EXTENSION_SOURCE_DIRECTORY: "./azure-iot-ops-cli-extension"
       K3S_VERSION: "v1.28.5+k3s1"
+      CA_FILE: "test-ca.pem"
+      CA_KEY_FILE: "test-ca-key.pem"
     strategy:
       fail-fast: false
       matrix:
-        feature: [default, mq-insecure, no-syncrules, ca-certs]
-        include:
-          # default / limited options
+        feature: [custom-input, default, mq-insecure, no-syncrules, ca-certs]
+        runtime-args:
+          - ${{ inputs.runtime-init-args != '' }}
+        exclude:
+          - feature: custom-input
+            runtime-args: false
           - feature: default
-            ca-valid-days: 3
-            kv-spc-secret-name: test-kv-secret
-          # test --mq-insecure deployment
+            runtime-args: true
           - feature: mq-insecure
-            mq-insecure: true
-            no-preflight: true
-            csi-config: 'telegraf.resources.limits.memory=500Mi telegraf.resources.limits.cpu=100m'
-          # test disabling custom sync rules
+            runtime-args: true
           - feature: no-syncrules
-            disable-rsync-rules: true
-          # test custom ca files
+            runtime-args: true
           - feature: ca-certs
-            ca-file: 'test-ca.pem'
-            ca-key-file: 'test-ca-key.pem'
+            runtime-args: true
     name: "Run cluster tests"
     runs-on: ubuntu-22.04
     steps:
+      - name: "Determine Init Args"
+        id: "init"
+        run: |
+          echo "NO_PREFLIGHT=false" >> $GITHUB_OUTPUT
+          if [[ ${{ matrix.feature }} == "default" ]]; then
+            echo "ARG=\"--ca-valid-days 3 --kv-spc-secret-name test-kv-secret\"" >> $GITHUB_OUTPUT
+          elif [[ ${{ matrix.feature }} == "mq-insecure" ]]; then
+            echo "ARG=\"--add-insecure-listener --csi-config telegraf.resources.limits.memory=500Mi telegraf.resources.limits.cpu=100m\"" >> $GITHUB_OUTPUT
```

Comment:
Please include a test run that has `--broker-service-type LoadBalancer` that can be combined with `--add-insecure-listener`.